doc/release/upcoming_changes/16987.improvement.rst

```diff
@@ -0,0 +1,14 @@
+Arbitrary `interval_size` option for ``numpy.unwrap``
+-----------------------------------------------------
+The size of the interval, over which phases are unwraped, is not restricted to `2 * pi` 
+anymore. This is especially useful for unwrapping degrees but can also be used for other
+intervals.
+
+.. code:: python
+
+    >>> phase_deg = np.mod(np.linspace(0,720,19), 360) - 180
```

Comment:
I think it would be helpful to show this too

Reply:
You mean an integer example?

Reply:
No, I mean print out the array

Reply:
```suggestion
    >>> phase_deg = np.mod(np.linspace(0,720,19), 360) - 180
    >>> phase_deg
    # whatever it is that prints out
```